elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/EmbeddedElasticsearchPolicy.java

```diff
@@ -114,15 +114,17 @@ void createIndex(String index, Map<String, String> mapping) throws IOException {
 
     ObjectNode mappings = mapper().createObjectNode();
 
-    ObjectNode properties = mappings.with("mappings").with(index).with("properties");
+    ObjectNode properties = mappings.with("mappings").with("properties");
     for (Map.Entry<String, String> entry: mapping.entrySet()) {
       applyMapping(properties, entry.getKey(), entry.getValue());
     }
 
     // create index and mapping
     final HttpEntity entity = new StringEntity(mapper().writeValueAsString(mappings),
         ContentType.APPLICATION_JSON);
-    restClient().performRequest("PUT", "/" + index, Collections.emptyMap(), entity);
+    final Request r = new Request("PUT", "/" + index);
```

Comment:
New API doesn't allow entity in constructor ?

Reply:
Entity is set in the next line. The many arguments versions of `performRequest` and `performRequestAsync` were dropped.
https://github.com/elastic/elasticsearch/pull/33223